client/components/jetpack/jetpack-product-info/style.scss

```diff
@@ -188,11 +202,38 @@
 	gap: 12px;
 }
 
+.jetpack-product-info__product-list-item-header {
+	display: flex;
+	flex-direction: column;
+
+	@media ( min-width: $break-medium ) {
+		flex-direction: row;
+		justify-content: space-between;
+		align-items: center;
+	}
+}
+
 .jetpack-product-info__product-list-item-title {
 	font-size: $font-body-small;
 	font-weight: 700;
 }
 
+.jetpack-product-info__product-list-item-callout {
+	padding: 0.25rem 0 0 0;
+	box-sizing: border-box;
+	color: var(--studio-gray-70);
+	font-size: 0.875rem;
+
+	@media ( min-width: $break-medium ) {
+		padding: 0.25rem;
+		min-width: 160px;
+		background-color: #eaefef;
```

Comment:
Can we use variable here?

Reply:
Good catch, @yashwin 👍 . Right now, this doesn't seem to have an existing variable. We will have to figure out where we can best put the variables so they can be reused, as there are a couple of colors we will also have to fix with similar situations. I am planning to fix it in the follow-up PR. For now, we can leave it as is.